pkg/plugins/plugindef/plugindef_types_gen.go

```diff
@@ -67,6 +67,7 @@ const (
 )
 
 // Defines values for Category.
+// Defines values for PlugindefCategory.
```

Comment:
yeah, yuck, something going there with the switch to dst or the adapted version of the logic, clearly 😢